### PR TITLE
:bug: 10941 - Keep the value for input number updating

### PR DESCRIPTION
**Pull request by waratah**
###Defect Fixes

10941

### CODE REVIEW

File: src/app/components/tabview/tabview.css [hunk 1]
```diff
@@ -6,6 +6,10 @@
     overflow: hidden;
 }
 
+.p-tabview-scrollable > .p-tabview-nav {
```
Comment (cetincakiroglu) [inline src/app/components/tabview/tabview.css:9]: why did you add this new css to tabview.css ?

**Review by cetincakiroglu — CHANGES_REQUESTED** [review]
Hi, what's the purpose of the changes in tabview.css, also could you please resolve conflicts?

File: src/app/components/tabview/tabview.css (hunk 1 above)
Comment (waratah) [reply src/app/components/tabview/tabview.css:9]: You are right there are overlayed changes.     From memory.  tabview.css changes were because there is a wrap layout definition that is not done properly.  there is a class to wrap the tabs however it is broken and does not actually wrap tabs when all the rest of the CSS works towards this.    This is from memory and it was over 12 months ago.

Comment (cetincakiroglu) [reply src/app/components/tabview/tabview.css]: Ok, reverted this changes.

**Review by cetincakiroglu — APPROVED** [review]

**Comment by cetincakiroglu** [comment]
Hi, 
could you please resolve conflicts so I can merge safely?

**Comment by mertsincan** [comment]
Hi,

Sorry for the delayed response! Unfortunately, your PR is out of date. So it's closed. Can you create a new PR for us? I will merge it as soon as possible.

Best Regards,
